quake-terminal@diegodario88.github.io/extension.js

```diff
@@ -63,6 +63,12 @@ export default class TogglerExtension extends Extension {
 			this._quakeMode._internalState === TERMINAL_STATE.DEAD
 		) {
 			const terminalId = this._settings.get_string("terminal-id");
+
+			if (!terminalId) {
+				Main.notify(_(`Select an application in Quake Terminal preferences.`));
+				return;
+			}
+
```

Comment:
Hey there! 

AFAIK  `terminal-id` will always have a value, since it has default properties on gschemas.

Reply:
In my tests I manually removed the value via dconf Editor and I noticed the message `No terminal found with id . Skipping ...`
So, from an abundance of caution, I thought that the users should get some instructions in case they messed up settings. Maybe it prevents issues being logged.
If you don't agree. I'll drop it.

Reply:
> In my tests I manually removed the value via dconf Editor and I noticed the message `No terminal found with id . Skipping ...` So, from an abundance of caution, I thought that the users should get some instructions in case they messed up settings. Maybe it prevents issues being logged. If you don't agree. I'll drop it.

How about changing the current message 'No terminal found...' to be more instructive?

Reply:
I think the message will be too long and get truncated. 
I'll logout for today. Please make whatever changes you like, if you want to get it done.

Cheers!

Reply:
> I think the message will be too long and get truncated. I'll logout for today. Please make whatever changes you like, if you want to get it done.
> 
> Cheers!

Alright, thanks for the initial work! Have a fantastic weekend 🙂